app/flows/student_finance_calculator_flow/outcomes/outcome_uk_full_time_dental_medical_students.erb

```diff
@@ -6,8 +6,8 @@
 
     In each of your first 4 years, you could get:
 
-    <%= render partial: 'full_time_tuition', locals: { tuition_fee_amount: calculator.tuition_fee_amount, maintenance_loan_amount: calculator.maintenance_loan_amount, maintenance_grant_amount: calculator.maintenance_grant_amount, start_date: calculator.course_start } %>
-
+    - <%= format_money(tuition_fee_amount, pounds_only: true) %> [Tuition Fee Loan](/student-finance/new-fulltime-students) to pay for your course
```

Comment:
TODO: in the commit message, link to the file / commit this content is copied from.